src/views/v2/emails/template-parts/body/order/ticket-totals/ticket-title.php

```diff
@@ -21,9 +21,17 @@
  * @var string                             $additional_content The email additional content.
  * @var bool                               $is_tec_active      Whether `The Events Calendar` is active or not.
  * @var \WP_Post                           $order              The order object.
+ * @var array<string, mixed>               $cart_item          The cart item.
  */
 
-$ticket_title = get_the_title( $ticket );
+use Tribe__Utils__Array as Arr;
+
+if ( empty( $cart_item['ticket_id'] ) ) {
+	return;
+}
+
+$fallback_title = get_the_title( $cart_item['ticket_id'] );
+$ticket_title = Arr::get( $cart_item, 'ticket_title', $fallback_title );
```

Comment:
I might be wrong, but I find the usage of classes within templates to build data on the fly is something we try to avoid, right?

Can't we have them as context variable?

Reply:
We should, but in this case trying to remove it will take more global changes that are not worth the time now.

Reply:
I agree with Rafsun.